.clang-tidy

```diff
@@ -1,81 +1,60 @@
+    -readability-function-cognitive-complexity,
+    -readability-implicit-bool-conversion,
+    -readability-implicit-bool-cast,
+    -readability-magic-numbers,
 
 CheckOptions:
-################## readability identifier naming ################################
-# class
-  - key:      readability-identifier-naming.ClassCase
-    value:    CamelCase
-  - key:      readability-identifier-naming.ClassMemberCase
-    value:    camelBack
-  - key:      readability-identifier-naming.ClassMemberPrefix
-    value:    _
-  - key:      readability-identifier-naming.ClassMethodCase
-    value:    camelBack
-  - key:      readability-identifier-naming.ClassConstantCase
-    value:    UPPER_CASE
-  - key:      readability-identifier-naming.ClassConstantCasePrefix
-    value:    _
-# enum
-  - key:      readability-identifier-naming.EnumCase
-    value:    CamelCase
-  - key:      readability-identifier-naming.ScopedEnumConstantCase
-    value:    UPPER_CASE
-  - key:      readability-identifier-naming.EnumConstantCase
-    value:    UPPER_CASE
-# others
-  - key:      readability-identifier-naming.FunctionCase
-    value:    camelBack
-  - key:      readability-identifier-naming.ConstantCase
-    value:    UPPER_CASE
-  - key:      readability-identifier-naming.GlobalConstantCase
-    value:    UPPER_CASE
-  - key:      readability-identifier-naming.LocalConstantCase
-    value:    camelBack
-  - key:      readability-identifier-naming.LocalVariableCase
-    value:    camelBack
-  - key:      readability-identifier-naming.MacroDefinitionCase
-    value:    UPPER_CASE
-  - key:      readability-identifier-naming.ParameterCase
-    value:    camelBack
-  - key:      readability-identifier-naming.StaticConstantCase
-    value:    UPPER_CASE
-  - key:      readability-identifier-naming.VariableCase
-    value:    camelBack
-  - key:      readability-identifier-naming.StructCase
-    value:    CamelBack
+    - { key: readability-identifier-naming.NamespaceCase,          value: lower_case }
+    - { key: readability-identifier-naming.StructCase,             value: CamelCase  }
+    - { key: readability-identifier-naming.ClassCase,              value: CamelCase  }
+    - { key: readability-identifier-naming.ClassMethodCase,        value: camelBack  }
+    - { key: readability-identifier-naming.ClassConstantCase,      value: UPPER_CASE }
+    - { key: readability-identifier-naming.ClassMemberCase,        value: camelBack  }
+    - { key: readability-identifier-naming.ProtectedMemberPrefix,  value: _          }
+    - { key: readability-identifier-naming.PrivateMemberPrefix,    value: _          }
+
+    - { key: readability-identifier-naming.EnumCase,               value: CamelCase  }
+    - { key: readability-identifier-naming.ScopedEnumConstantCase, value: UPPER_CASE }
+    - { key: readability-identifier-naming.EnumConstantCase,       value: UPPER_CASE }
 
+    - { key: readability-identifier-naming.FunctionCase,           value: aNy_CasE   }
```

Comment:
Why function name is `aNy_CasE`?

Reply:
Google uses the loose requirement on this. We have many functions that are not strictly following camelBack style, so I think for now we can hold on to this rule?

Reply:
If using this rule, then `Class::Func()` will be valid?

Reply:
Yes

Reply:
I am agree we can hold on this rule and add some comments for existing codes, but then we can not check new codes.